packages/core/src/Constants/cfd-text.js

```diff
@@ -21,4 +21,5 @@ export const CFD_TEXT = {
     all_zero_spread_demo: () => localize('Zero Spread Demo'),
     all_swap_free_svg: () => localize('Swap-Free SVG'),
     all_zero_spread_bvi: () => localize('Zero Spread BVI'),
+    gold: () => localize('Gold'),
```

Comment:
[Q] not financial_gold?

Reply:
i am followed the `stp` since both of them kind of have same condition